docs/rules/require-return-from-computed.md

```diff
@@ -24,6 +24,13 @@ export default Component.extend({
       return value;
     }
   }),
+  
+  salutation: computed('firstName', function() {
+    if (this.get('firstName')) {
+      return `Dr. ${this.get('firstName')}`
+    }
+    return;
```

Comment:
Would you be able to switch this to `return ''`, `return null`, or `return undefined`? That way, it would comply with the [consistent-return](https://eslint.org/docs/rules/consistent-return) rule.